### DISCUSSION TITLE
IE11 and SCRIPT28: Out of stack space

### DISCUSSION THREAD

**Discussion by scil** [zloirock/core-js#1113] [General]
I use 'core-js-bundle/index.js` and encounter this error with IE11
```
SCRIPT28: Out of stack space
core-js-bundle.js (1035,3)
```

the location is 
```
// `RequireObjectCoercible` abstract operation
// https://tc39.es/ecma262/#sec-requireobjectcoercible
module.exports = function (it) {
  if (it == undefined) throw $TypeError("Can't call method on " + it);
  return it;
};

```

src is [require-object-coercible.js](https://github.com/zloirock/core-js/blob/6870c2ad4e896f0350c92a4e7b729694b5841766/packages/core-js/internals/require-object-coercible.js)

At first, I thought the error is about core-js, but after tracing back, I went down to another lib [pjax-api/](https://github.com/falsandtru/pjax-api/) which declares no ie support.

just share my experience.

hope it helpful for someone else.

**Comment by zloirock** [comment]
Most likely, the issue is in a bug in the way of transpiling / polyfills injection. Without a reproducible example, I can't say more.

**Reply by zloirock** [reply]
Also, please, next time open such issues as issues, not discussions.

**Reply by scil** [reply]
Thank you for your reply.

I dont think it's a bug. because [pjax-api/](https://github.com/falsandtru/pjax-api/) does not support IE.

**Reply by zloirock** [reply]
Maybe this library doesn't support IE, but I am 90% sure that the error that you described is a bug in the way of transpiling - just enable logic - do you see something that could call `Out of stack space` error in the source of this file? It looks like circular dependencies.

Try to exclude `core-js` path from transpiling in your transpiler / bundler config.

**Reply by scil** [reply]
Thank you.

I use `<script src="https://cdn.jsdelivr.net/npm/core-js-bundle"></script>`  for  only IE.

before I use babel to transform [pjax-api/](https://github.com/falsandtru/pjax-api/), now I exclude it.

**Reply by zloirock** [reply]
So, it seems the function / module where this error happend localized not correctly. Anyway, it's not a `core-js` issue.

**Reply by scil** [reply]
I think so too.  the transformed pjax-api just made this module executing repeatly
```
module.exports = function (it) {
  if (it == undefined) throw $TypeError("Can't call method on " + it);
  return it;
};
```
